core/src/main/scala/org/apache/spark/status/AppHistoryServerPlugin.scala

```diff
@@ -35,4 +35,11 @@ private[spark] trait AppHistoryServerPlugin {
    * Sets up UI of this plugin to rebuild the history UI.
    */
   def setupUI(ui: SparkUI): Unit
+
+  /**
+   * Order of the plugin tab that need to display in the history UI.
```

Comment:
The position of a plugin tab relative to the other plugin tabs in the history UI.

Reply:
Done